mockgen/internal/tests/defined_import_local_name/input.go

```diff
@@ -0,0 +1,13 @@
+package defined_import_local_name
+
+import (
+	"bytes"
+	"context"
+)
+
+//go:generate mockgen -package defined_import_local_name -destination mock.go -source input.go -imports b_mock=bytes,c_mock=context
+
+type WithDotImports interface {
```

Comment:
What is DotImports referring to? In the spec I see three cases
```
Import declaration          Local name of Sin

import   "lib/math"         math.Sin
import m "lib/math"         m.Sin
import . "lib/math"         Sin
```

As a reviewer, the DotImport name makes me think this is that last case in the list, but this test seems to be testing named imports (case 2).